common/static/common/js/components/views/feedback.js

```diff
@@ -4,7 +4,8 @@
             "underscore",
             "underscore.string",
             "backbone",
-            "text!common/templates/components/system-feedback.underscore"],
+            "text!common/templates/components/system-feedback.underscore",
+            "jquery.ui.selectable"],
```

Comment:
@andy are you suggesting to upgrade so that the feature is there, and then use the whole library? Or upgrade it for other places that already use the whole thing and then include the smaller package here, knowing that it won't conflict?  I could also downgrade the selectable portion to 1.10.4.